src/pages/mobile/Transfer.vue

```diff
@@ -124,6 +124,9 @@ export default {
         }] : [], {
           selector: '.transfer .tab-bar .button-plain:nth-child(3)',
           ...this.$t('transfer.tooltips.account-switcher'),
+        }, {
+          selector: '.transfer .wrapper .list-item:nth-child(5)',
```

Comment:
I propose to add an additional class to the list item pointing to redeem. Selecting by position won't work in this case because we have a different amount of items on Android and iOS.